android/settings.gradle

```diff
@@ -1,6 +1,20 @@
 rootProject.name = 'weex-project'
+
+def updateSubmodule(){
```

Comment:
Unless you have a special reason, please use groovy task for better build speed. 

```
task updateSubmodules(type: Exec){
    description 'Update git submodule'
    commandLine 'git submodule update --init --remote'.split()
}
```

And find somewhere to execute the task